app/src/docs/_examples/tables/FiltersPanelBasic.example.tsx

```diff
@@ -82,6 +82,7 @@ export default function FiltersPanelExample() {
                 columnKey: 'jobTitle',
                 title: 'Title',
                 type: 'multiPicker',
+                maxCount: 2,
```

Comment:
it's no sense to pass maxCount: 2, because 2 is a default value. Better lets add maxCount: 3, to the 'profileStatus' filter, because in this case we have short names and only 3 options.